### ISSUE TITLE
Load bible texts directly from SWORD modules (instead from the database)

### ISSUE THREAD

**Issue by tobias-klein** [tobias-klein/ezra-project#28]
**Background**
Ezra Project has been based on a concept where the bible text is stored in a database. This has historic reasons, Ezra Project used to be a (Ruby on Rails) web application with a server-side backend and data stored in a MySQL database.

**Refactoring**
Ezra Project shall be refactored to access SWORD modules directly for all usecases (load the text from the locally available SWORD modules). Once this refactoring is done a migration shall change the existing databases to not store bible data any longer.

Removing the "sync features" (syncing translations between local sword modules and Ezra db) is handled in a separate issue #29.

**Comment by alerque** [comment]
This is great and works like a charm for me, good work!

**Comment by tobias-klein** [comment]
Thanks. I'm glad to hear that! :smile:
